### PR TITLE
Fixed testcases of admin- and snippet-bundle for symfony 3.0

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | is a part of #2106 
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR fixes testcases of admin- and snippet-bundle for symfony 3.

* AdminBundle
* SnippetBundle

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Tests/app/config/config.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ security:
                 check_path: sulu_admin.login_check
                 success_handler: sulu_security.authentication_handler
                 failure_handler: sulu_security.authentication_handler
-                csrf_provider: security.csrf.token_manager
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Tests/app/config/config.yml]: Will that also be removed in sulu-standard and sulu-minimal? Is the CSRF protection still working?

File: src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php [hunk 2]
```diff
@@ -97,7 +97,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $overwrite = $input->getOption('overwrite');
         $dryRun = $input->getOption('dry-run');
 
-        $this->session = $this->getContainer()->get('doctrine_phpcr')->getManager()->getPhpcrSession();
+        $this->session = $this->getContainer()->get('doctrine_phpcr.session');
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php]: Actually we should inject the correct session here... But I think this command is not working correctly at the moment, since it looks to me as if the changes are not transferred to the live workspace with a `publish` call...

File: src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php [hunk 3]
```diff
@@ -86,6 +86,9 @@ public static function dataFieldsProvider()
                         'search' => 'now',
                         'searchFields' => 'title',
                         'page' => 1,
+                    ],
+                    [],
+                    [
```
Comment (danrot) [inline src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php]: Why is that necessary, and why was it changed only for this request?

File: src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php [hunk 4]
```diff
@@ -59,99 +54,118 @@ private function getGetBooleanRequestParameterReflection()
 
     public function testGetRequestParameter()
     {
-        $request = $this->getRequestMock(
-            [
-                ['test', null, false, 'data'],
-                ['none', 'default', false, 'default'],
-            ]
-        );
+        $request = $this->prophesize(Request::class);
```
Comment (danrot) [inline src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php]: Why not simply using the real Request object? That's also what the prophecy team suggests: https://github.com/phpspec/prophecy/issues/86#issuecomment-64948339

File: src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php [hunk 5]
```diff
@@ -59,99 +54,118 @@ private function getGetBooleanRequestParameterReflection()
...
-                ['test1', null, false, 'true'],
-                ['test2', null, false, 'false'],
-                ['none', 'default', false, null],
-            ]
-        );
+        $request = $this->prophesize(Request::class);
+        $request->get('test1', null)->willReturn('true');
+        $request->get('test2', null)->willReturn('false');
+        $request->get('none', null)->willReturn(null);
+        $request->get('none', true)->willReturn(true);
```
Comment (danrot) [inline src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php]: Same as above

File: src/Sulu/Bundle/AdminBundle/Tests/app/config/config.yml (hunk 1 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/AdminBundle/Tests/app/config/config.yml]: in sulu-standard we have to change following: https://github.com/sulu/sulu-standard/pull/770/files#diff-3a35616aea3903a1edf9541042da2b51R36

in the tests we havent used it so i have removed it.

File: src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php (hunk 2 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php]: i have created an issue for that https://github.com/sulu/sulu/issues/3079

File: src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php (hunk 3 above)
Comment (wachterjohannes) [inline src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php]: in the test above it also has to be done. the function `Request::getRequestFormat()` in 3.0 searches in the attributes for `_format`. maybe in the testcase before the format wont be requested. but i will check this.

**Comment by wachterjohannes** [comment]
@danrot fixed

File: src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php (hunk 4 above)
Comment (wachterjohannes) [inline src/Sulu/Component/Rest/Tests/Unit/RequestParametersTraitTest.php]: but now (:
